datasets/chorus-dt/scripts/resolve_places.py

```diff
@@ -87,6 +104,9 @@ def compute_distance(x):
             "trip_place_%d" % place_index
         ].apply(lambda x: places_dict["total"][x])
 
-    trips["distance"] = trips.apply(compute_distance, axis=1)
+    trips["distance_1"] = compute_distance(trips, 1)
```

Comment:
I find this one a bit blurry, might be more readable to have something like:
```suggestion
    trips["distance_1"] = trips[['place_0_coords', 'place_1_coords']].apply(compute_distance, axis=1)
        trips["distance_2"] = trips[['place_1_coords', 'place_2_coords']].apply(compute_distance, axis=1)
```

But might be done later.

Reply:
This entails modifying the compute_distance function. Will do it since the whole process would be much clearer after.